linera-core/src/data_types.rs

```diff
@@ -237,6 +237,10 @@ pub enum CrossChainRequest {
     },
 }
 
+/// An discriminant for message queues: messages with the same queue ID will be delivered
+/// in order.
+pub type CrossChainQueueId = (ChainId, ChainId, bool);
```

Comment:
`pub type` aliases are a bit meh, so are tuples. Can we have a struct? Does it need to be public?